Library/Homebrew/formula_installer.rb

```diff
@@ -205,8 +205,15 @@ def install
     oh1 "Installing #{Tty.green}#{formula.full_name}#{Tty.reset}" if show_header?
 
     if formula.tap && !formula.tap.private?
-      options = effective_build_options_for(formula).used_options.to_a.join(" ")
-      Utils::Analytics.report_event("install", "#{formula.full_name} #{options}".strip)
+      options = effective_build_options_for(formula).used_options.to_a
+      if formula.head?
+        options << "--HEAD"
+      elsif formula.devel?
+        options << "--devel"
+      end
```

Comment:
I think technically this part is fine. I wonder though:

- Should we prefix this to the options instead of appending it? If other options are involved, this will give a more uniform look as all list entries will start with `<formula> --devel` (e.g. for the devel spec).
- Does it make sense to have a completely separate category (e.g. `install/devel` and `install/head`) instead of mixing them into the big bowl of regular installs? Or is the same category fine and the analytics UI does a good enough job of filtering the entries for us afterwards?